### PR TITLE
Add Many Descriptions To The `Window` Class

### PR DESCRIPTION
**Pull request by SirQuartz**
Added several missing descriptions for various functions and properties. I don't fully understand what every function of this node does yet, but I think I'm on the right track with these additions after some testing. Plus it's better than nothing. I intend to improve this wherever possible.

If you've any feedback, recommendations, and/or insights I'd be glad to hear it.

### CODE REVIEW

File: doc/classes/Window.xml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Window" inherits="Viewport" version="4.0">
 	<brief_description>
+		A general-purpose sub-window.
```
Comment (akien-mga) [inline doc/classes/Window.xml:4]: It's not necessarily a sub-window, as the main window is also a `Window`.

File: doc/classes/Window.xml [hunk 2]
```diff
@@ -38,6 +42,7 @@
 			<return type="Vector2i">
 			</return>
 			<description>
+				Returns the window's size including it's border.
```
Comment (akien-mga) [inline doc/classes/Window.xml:45]: *its

File: doc/classes/Window.xml [hunk 3]
```diff
@@ -106,12 +111,14 @@
 			<return type="void">
 			</return>
 			<description>
+				Causes the window to attain focus, allowing it to receive user-input.
```
Comment (akien-mga) [inline doc/classes/Window.xml:114]: I'm not sure "attain focus" is correct, sounds mystical :) ("attain nirvana")

"grab" or "take" would be clearer and easier on non-native speakers.

File: doc/classes/Window.xml [hunk 4]
```diff
@@ -11,6 +13,7 @@
 			<return type="bool">
 			</return>
 			<description>
+				Returns [code]true[/code] if the window is being drawn to the screen.
```
Comment (akien-mga) [inline doc/classes/Window.xml:16]: We tend to use "Returns whether" instead of "Returns true if".

File: doc/classes/Window.xml [hunk 5]
```diff
@@ -325,6 +336,7 @@
 		<member name="content_scale_size" type="Vector2i" setter="set_content_scale_size" getter="get_content_scale_size" default="Vector2i( 0, 0 )">
 		</member>
 		<member name="current_screen" type="int" setter="set_current_screen" getter="get_current_screen" default="0">
+			The screen the window will appear on if the user has more than one display.
```
Comment (akien-mga) [inline doc/classes/Window.xml:339]: Not only "will appear on", the API doesn't necessarily describe only what happens after you create a node, but also the current state. So this is the screen the window **is** displayed on too.

File: doc/classes/Window.xml [hunk 6]
```diff
@@ -333,16 +345,21 @@
 		<member name="min_size" type="Vector2i" setter="set_min_size" getter="get_min_size" default="Vector2i( 0, 0 )">
 		</member>
 		<member name="mode" type="int" setter="set_mode" getter="get_mode" enum="Window.Mode" default="0">
+			The window's current mode (i.e windowed, fullscreen, minimized, etc).
```
Comment (akien-mga) [inline doc/classes/Window.xml:348]: `i.e.` and `etc.`

**Comment by SirQuartz** [comment]
@akien-mga Well, this branch had conflicts, and apparently, I ended up deleting it a while ago, so not really sure what to do with the pull request. Make a new one?

**Comment by akien-mga** [comment]
> Make a new one?

Yes that would be needed then. You can download the changes with https://github.com/godotengine/godot/pull/49344.patch and apply them on an up-to-date branch with `git am -3 path/to/49344.patch`. Then you'll have the conflicts to resolve, `git add` once fixed and `git am --continue` to finish applying the patch with the conflicts solved.

**Comment by akien-mga** [comment]
Superseded by #50182.

**Pull request closed without merging**